code/controllers/subsystem/processing/circuit.dm

```diff
@@ -0,0 +1,23 @@
+PROCESSING_SUBSYSTEM_DEF(circuit)
+	name = "Circuit"
+	stat_tag = "CIR"
+	var/list/all_exonet_connections = list()
+	var/list/all_integrated_circuits = list()
+	var/cipherkey
+	init_order = INIT_ORDER_CIRCUIT
+	flags = SS_BACKGROUND|SS_POST_FIRE_TIMING
```

Comment:
is this a really expensive thing or something this basicall gives it lowest priority and makes it lag if the server is lagging at all.